### ISSUE TITLE
Feature request: make plugin work when Laravel App is served from a different domain/server

### ISSUE THREAD

**Issue by lucaspottersky** [NoelDeMartin/cypress-laravel#5]
### Problem:
In setups where the PHP app backend is being served from a different server than where the the one where frontend is running (say an API at `http://backend.localhost`), the plugin throws exceptions:

> CypressError: `cy.request()` failed on:
> 
> http://frontend.localhost/_cypress/setup
> 

### Suggested solution:
There could be some sort of setting where you could specify to override the "baseUrl". For example, in `cypress.json`

```
env: {
  cypress-laravel-baseUrl: "http://backend.localhost"
}
```

When none is specified, than follow the current behaviour.

**Comment by NoelDeMartin** [comment]
Hey thanks for the suggestion. It sounds reasonable, I'll look into it 👍.

**Comment by NoelDeMartin** [comment]
I've released a new version (0.2.0) with support for this, thanks for your contribution in the PR!

I've also added a [decoupled frontend](https://github.com/NoelDeMartin/laravel-cypress-sandbox/tree/master/frontend) with tests in the sandbox project, in case you want to see how I've configured it.
